app.rb

```diff
@@ -203,18 +203,7 @@ def soft_redirect(url, page_title)
 end
 
 get '/needed.html' do
-  if (token = ENV['GITHUB_ACCESS_TOKEN']).to_s.empty?
-    warn 'No GITHUB_ACCESS_TOKEN found'
-    client = Octokit::Client.new
-  else
-    client = Octokit::Client.new(access_token: token)
-  end
-  client.auto_paginate = true
-
-  @to_find   = client.issues 'everypolitician/everypolitician-data', labels: 'New Country,To Find'
-  @to_scrape = client.issues 'everypolitician/everypolitician-data', labels: 'New Country,To Scrape'
-  @to_finish = client.issues 'everypolitician/everypolitician-data', labels: 'New Country,3 - WIP'
-
+  @page = Page::Needed.new(ENV['GITHUB_ACCESS_TOKEN'])
```

Comment:
I have a very strong suspicion that we should always be passing named parameters (or Hash), rather than a list of contextless arguments. We don't necessarily need to do that in this PR; more just capturing the thought here…